server/webdriver/tests/browser_test.py

```diff
@@ -255,7 +255,8 @@ def test_stat_var_hierarchy(self):
     WebDriverWait(self.driver, self.TIMEOUT_SEC).until(loading_finished)
     first_result = self.driver.find_element(
         By.XPATH,
-        '//*[@id="stat-var-hierarchy-section"]/div[1]/div[1]/div[2]/div/div[1]')
+        '//*[@id="stat-var-hierarchy-section"]/div[1]/div[1]/div[1]/div/div[2]/div[1]'
```

Comment:
Possible refactor: `'//*[@id="stat-var-hierarchy-section"]//div[contains(@class, "statvar-hierarchy-search-results")]'`

Reply:
Thanks for the tip! Refactored.